Header header
uint32 event_id
geometry_msgs/Point location
float32 area
float32 confidence